src/openmc_heat_driver.cpp

```diff
@@ -180,16 +181,20 @@ void OpenmcHeatDriver::init_mappings()
 
 void OpenmcHeatDriver::init_tallies()
 {
+  using gsl::index;
```

Comment:
What do you think about putting this in the file scope?  Then you could clean up all the other functions, too, and it would all read consistently.

Reply:
Good idea; I'll change that